src/components/behandlerdialog/meldingtilbehandler/MeldingsTypeInfo.tsx

```diff
@@ -0,0 +1,58 @@
+import { MeldingType } from "@/data/behandlerdialog/behandlerdialogTypes";
+import styled from "styled-components";
+import React from "react";
+import {
+  BlueDocumentImage,
+  BlyantImage,
+} from "../../../../img/ImageComponents";
+import { BodyShort } from "@navikt/ds-react";
+
+const text = {
+  tilleggsopplysinger:
+    "Tilleggsopplysninger vedørende pasienten. Behandleren honoreres med takst L8.",
+  legeerklaring:
+    "Legeerklæring vedørende pasienten. Behandleren honoreres med takst L46.",
+};
+
+const Icon = styled.img`
+  margin-right: 1em;
+  width: 3em;
+`;
+
+const InfoPanel = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+interface Props {
+  meldingType: MeldingType;
+}
+
+export const MeldingsTypeInfo = ({ meldingType }: Props) => {
+  const Info = () => {
+    switch (meldingType) {
+      case MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER:
+        return (
+          <>
+            <Icon src={BlyantImage} />
+            <BodyShort size={"small"}>{text.tilleggsopplysinger}</BodyShort>
+          </>
+        );
+      case MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING:
+        return (
+          <>
+            <Icon src={BlueDocumentImage} />
+            <BodyShort size={"small"}>{text.legeerklaring}</BodyShort>
+          </>
+        );
+      case MeldingType.FORESPORSEL_PASIENT_PAMINNELSE:
+        return <></>; // Not supported
```

Comment:
Vi kan bruke `default` her i stedet: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Ville kanskje trodd IntelliJ sa ifra om det, men gjør kanskje ikke det fordi vi har håndtert alle (de tre) casene allerede 🤔 Så det er kanskje ikke så farlig 🤷🏼‍♂️

Reply:
Jeg synes det er digg å kjøre exhaustive case match i stedet for en catch all, med mindre det er fåtallet som skal håndteres. Når/hvis det legges til en ny type blir vi da nødt å ta stilling til om vi vil håndtere det eller ikke 😊
Men hvis vi ikke får warning her så hjelper jo det jeg sier fint lite🙃🤔 Det høres ut som noe vi bør ha i linting reglene våre😁